src/main/resources/META-INF/rewrite/ibm-java.yml

```diff
@@ -110,4 +112,34 @@ recipeList:
   - org.openrewrite.java.ChangeType:
       oldFullyQualifiedTypeName: com.ibm.security.auth.module.Krb5LoginModule
       newFullyQualifiedTypeName: com.sun.security.auth.module.Krb5LoginModule
-      ignoreDefinition: true
\ No newline at end of file
+      ignoreDefinition: true
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.RemovedJavaXMLWSModuleProvided
+displayName: Do not use `java.xml.ws` module in WebSphere Liberty
+description:
+  The `java.xml.ws` module was removed in Java11. Websphere Liberty provides its own implementation of the module, which can be used by specifying the `jaxws-2.2` feature in the server.xml file.
+  This recipe removes the `javax.xml.ws` module from the application's build dependency in favor of the Websphere Liberty implementation to avoid class loading issues.
+tags:
+  - java11
+recipeList:
+  - org.openrewrite.java.dependencies.RemoveDependency:
+      groupId: javax.xml.ws
+      artifactId: jaxws-api
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.RemovedJaxBModuleProvided
+displayName: Do not use `java.xml.bind` and `java.activation` modules in WebSphere Liberty
+description:
+  The `java.xml.bind` and `java.activation` module were removed in Java11.
+  Websphere Liberty provides its own implementation of the module, which can be used by specifying the `jaxws-2.2` feature in the server.xml file.
```

Comment:
Websphere Liberty provides its own implementation of **these modules**...
Also, the feature here should `jaxb-2.2` not `jaxws-2.2`

Reply:
updated in next commit